compiler/codegen/symbol.cpp

```diff
@@ -60,16 +60,32 @@
 *                                                                   *
 ********************************* | ********************************/
 
-char llvmFuncDumpName[FUNC_NAME_MAX+1] = "";
-const char *llvmFuncDumpCName = "";
-int llvmFuncOptDump = 0;
+char llvmPrintIrName[FUNC_NAME_MAX+1] = "";
+char llvmPrintIrStage[FUNC_NAME_MAX+1] = "";
+
+int llvmPrintIrNameStageNum;
+const char *llvmPrintIrCName;
+
+std::map<std::string, int> llvmStageMap =
+{
+    {LLVM_NONE_STAGE_NAME, LLVM_NONE_STAGE_NUM},
+    {LLVM_BASIC_STAGE_NAME, LLVM_BASIC_STAGE_NUM},
+    {LLVM_FULL_STAGE_NAME, LLVM_FULL_STAGE_NUM}
+};
+
+std::map<int, std::string> llvmStageRevMap =
```

Comment:
std::map seems overkill for these 3-element debugging lists. I'd personally probably just use an array for int -> string and then linear search for the other way... but there's not anything wrong exactly with using std::map.